cmd/template/cluster/provider/templates/aws/aws_cluster.yaml.tmpl

```diff
@@ -0,0 +1,19 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
+kind: AWSCluster
+metadata:
+  labels:
+    cluster.x-k8s.io/cluster-name: {{ .Name }}
```

Comment:
I am thinking of having these labels somehow applied in a patch - doing a dumb copy paste for now.